### DISCUSSION TITLE
WiKi

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#444] [Polls]
первый пост

**Comment by Kor-Angar** [comment]
Ягоды гоблина
 
@dosinabox подскажи пожалуйста у кого, сколько и в какой главе появляются в продаже ДОПОЛНИТЕЛЬНЫЕ ягоды гоблина.

**Reply by Kor-Angar** [reply]
upd ДОПОЛНИТЕЛЬНЫЕ ягоды гоблина в продаже

**Reply by dosinabox** [reply]
Мигель - 1 штука - 1 глава
Энгор - 1 штука - 1 глава
Сагитта - 1 штука - 3 глава
Кронос - 1 штука - 4 глава

**Reply by Kor-Angar** [reply]
Если выпадание ягод гоблина из паладинов отключено (в scriptpatch отключено по умолчанию), то у торговцев появятся дополнительные 4 ягоды гоблина в продаже:

Энгор - Миненталь
Мигель - Яркендар
Сагитта - Хоринос(3 глава)
Кронос - Яркендар(4 глава)

https://github.com/dosinabox/g2nr_unofficial_update/wiki/Ягоды-Гоблина

**Reply by Kor-Angar** [reply]
@dosinabox у тебя стоимость ягод всё так же увеличина до 500 ?

**Reply by dosinabox** [reply]
250

**Comment by Kor-Angar** [comment]
Проверь пожалуйста цифры [Ghjdthm](https://github.com/dosinabox/g2nr_unofficial_update/wiki/Orc-Elite) а то я подзапуталя

**Comment by Kor-Angar** [comment]
проверь правильное я прочитал твои проверки https://github.com/dosinabox/g2nr_unofficial_update/wiki/%22Зло%22---враги-Паладинов
